hooks/pre-commit

```diff
@@ -13,3 +13,25 @@ then
   echo Please commit to a working branch and push it to GitHub for review/merge.
   exit 1
 fi
+
+readonly changed_files_command="git diff --cached --name-only | grep '\.py$' | grep -v 'servers/[^\/]*/jobs/models' | grep -v 'setup.py'"
+readonly files_in_commit=$(eval $changed_files_command)
+echo $files_in_commit
```

Comment:
I prefer not to have all these echo lines (line 19, 22, 28) for every commit.

Ideally precommit would automatically format and save files, without manual intervention, like "pretty-quick --staged". Since this PR doesn't do that, I would like to either cancel this PR, or remove all this logging.

Reply:
Done